lib/railway_ipc/core/event_message.ex

```diff
@@ -11,14 +11,14 @@ defmodule RailwayIpc.Core.EventMessage do
     |> decode()
   end
 
-  def decode(%{encoded_message: encoded_message} = command_message) do
+  def decode(%{encoded_message: encoded_message} = event_message) do
```

Comment:
nice naming here.

Reply:
95% sure the original bad naming was due to copy/paste